stylix/mk-target.nix

```diff
@@ -0,0 +1,102 @@
+{
+  name,
+  humanName,
+  autoEnable ? true,
+  extraOptions ? { },
+  configElements ? [ ],
+  generalConfig ? null,
+}:
+{ config, lib, ... }:
+let
+  cfg = config.stylix.targets.${name};
+in
+{
+  options.stylix.targets.${name} = lib.attrsets.unionOfDisjoint {
+    enable = config.lib.stylix.mkEnableTarget humanName autoEnable;
+  } extraOptions;
+
+  config =
+    let
+      args =
```

Comment:
This name is confusing. What does the function do and/or what is the motivation for using it? Maybe answering that will inspire a more descriptive name :crossed_fingers: 

If we can't find a descriptive enough name that the function's purpose is obvious, then maybe a comment is needed too?

The confusion is made worse by there being an `args` variable shadowing this name further down :grin: